### ISSUE TITLE
Update readme to include slides

### ISSUE THREAD

**Issue by fatecabag** [mozillaph/advocacy#8]
### Description:

Update readme.md to include slide decks used during presentations. Please add slide links to this issue so we can add it to the readme
### Related Issues & Links (if any):

Issues: #
[Links](url)

**Comment by fatecabag** [comment]
@ryanermita maybe you can help in this?

**Comment by ryanermita** [comment]
I think it would be great and organize if we use wiki for event documentation and Readme for the overview of this repo.

**Comment by ryanermita** [comment]
For the event documentation maybe we can use this format.

### Event Title

description here ...

#### Report

link to event report

#### Resources
- resources 1
- resources 2
- resource 3

**Comment by fatecabag** [comment]
I was thinking of a blog post for event documentation and a wiki for all the resources. Any insights for this @fayetandog

**Comment by fayepal** [comment]
Good idea. Not sure if wiki and Github, kinda redundant? Maybe choose just one

**Comment by ryanermita** [comment]
What i meant is [github wiki](https://help.github.com/articles/about-github-wikis/) :)
We can use the format above or create a github wiki page for our slide decks :)
